lib/bcdice/game_system/BattleTech.rb

```diff
@@ -26,24 +28,45 @@ class BattleTech < Base
         　　正面からダメージ3の攻撃を技能ベース2目標値4で1回判定
         　例）2BT3RL+5>=8
         　　右下半身にダメージ3の攻撃を技能ベース5目標値8で2回判定
-        　ミサイルによるダメージは BT(ダメージ)の変わりに SRM2/4/6, LRM5/10/15/20を指定
+        　ミサイルによるダメージは BT(ダメージ) の代わりに SRM2/4/6, LRM5/10/15/20 を指定
         　例）3SRM6LU+5>=8
         　　左上半身にSRM6連を技能ベース5目標値8で3回判定
+          BT(ダメージ) の代わりに PPC を指定するとダメージ10で判定
+          例）2PPCR+3>=10
+          　右側からPPC（ダメージ10）による攻撃を技能ベース3目標値10で2回判定
         ・CT：致命的命中表
         ・DW：転倒後の向き表
         ・CDx：メック戦士意識維持表。ダメージ値xで判定　例）CD3
       MESSAGETEXT
 
-      register_prefix('\d*SRM', '\d*LRM', '\d*BT', 'PPC', 'CT', 'DW', 'CD')
+      register_prefix('\d*SRM', '\d*LRM', '\d*BT', '\d*PPC', 'CT', 'DW', 'CD')
 
       # 致命的命中が発生しない上限値
       NO_CRITICAL_HIT_LIMIT = 7
 
+      # @return [Command::Parser] PPCコマンドの構文解析器
+      def self.ppc_parser
+        return @ppc_parser if @ppc_parser
+
+        @ppc_parser = Command::Parser.new(/PPC(?:[LCR][LU]?)?/, round_type: :floor)
```

Comment:
`RoundType`で指定するのがいいと思います
```suggestion
        @ppc_parser = Command::Parser.new(/PPC(?:[LCR][LU]?)?/, round_type: RoundType::FLOOR)
```

Reply:
あっ、定数がありましたね。こちらに変えさせていただきます。